### ISSUE TITLE
Desktop - Account Verification - Unable to request resend of code for email verification

### ISSUE THREAD

**Issue by oxtoacart** [getlantern/lantern#3927]
If I enter the wrong number, or didn't get my SMS for some reason, I'm out of luck because the Mobile Phone Number input is no longer active.  I have to exit the flow and start over.

Especially since we have the text `Didn't receive a code? Please check your number and try again`, it seems that we really ought to give you some way to try again.

**Comment by sunshine19090** [comment]
@Derekf5 @p3rcya @oxtoacart NA on phone number basis. Should we have something similar in case of email verification while authorizing?

**Comment by p3rcya** [comment]
Yeah, users should be able to modify and resend email in case they entered the wrong email too.

**Comment by sunshine19090** [comment]
@oxtoacart @p3rcya Modified the description

**Comment by Derekf5** [comment]
I agree @sunshine19090 @p3rcya but i think @myleshorton said this wouldnt make it into the MVP

**Comment by sunshine19090** [comment]
I think that is okay @Derekf5. It is not a blocker or a issue really.  We should move it to a different milestone to keep track of the things left out in MVP.

**Comment by Derekf5** [comment]
cool thanks @sunshine19090!
